ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml

```diff
@@ -1,31 +1,5 @@
 postsubmits:
   openshift/ci-tools:
```

Comment:
Same changes are added here: https://github.com/openshift/release/pull/42830/files#diff-6f4b8fe33317cf6232786a05c09f7ee174c2d3efa03b700eb90b4fabb4fc7da9

Could we please rebase this PR on master.